tools/check_test.js

```diff
@@ -30,6 +30,12 @@ function Driver() {
     fail: 0,
     skip: 0,
     timeout: 0,
+    json: {
+      bin: { text:0, total:0, data:0, bss:0, rodata:0 },
```

Comment:
@robertsipka This json would be sent to be stored. Can you clarify where we handle the above data? We may need more information like commit hash to identify the base code of each result.